frontend/src/components/Schedules/PanelVariableOption.tsx

```diff
@@ -35,41 +42,45 @@ export const PanelVariableOptions: FC<Props> = ({
 
   return (
-    <div style={{ display: 'flex', flexDirection: 'row', marginTop: '5px' }}>
+    <div style={{ display: 'flex', flexDirection: 'row', marginTop: '5px', flexWrap: 'wrap' }}>
       <InlineFormLabel>{name}</InlineFormLabel>
-      {!multiSelectable ? (
-        <Select
-          value={options?.filter((f: any) => !!selectedOptions?.find((s1: any) => s1 === f.value.value))}
-          onChange={(selected: SelectableValue<SelectableVariable>) => {
-            if (selected.value?.value === '$__all') {
-              return onUpdate(selectableOptions);
-            }
+      <div className={flexContainer}>
+        {!multiSelectable ? (
+          <Select
+            value={options?.filter((f: any) => !!selectedOptions?.find((s1: any) => s1 === f.value.value))}
+            onChange={(selected: SelectableValue<SelectableVariable>) => {
+              if (selected.value?.value === '$__all') {
+                return onUpdate(selectableOptions);
+              }
 
-            onUpdate([selected]);
-          }}
-          options={options}
-        />
-      ) : (
-        <MultiSelect
-          onChange={(selected: SelectableValue<SelectableVariable>) => {
-            const isAll = selected.some(({ value }: SelectableValue<SelectableVariable>) => value?.value === '$__all');
-            if (isAll) {
-              return onUpdate(selectableOptions);
-            }
+              onUpdate([selected]);
+            }}
+            options={options}
+          />
+        ) : (
+          <MultiSelect
+            onChange={(selected: SelectableValue<SelectableVariable>) => {
+              const isAll = selected.some(
+                ({ value }: SelectableValue<SelectableVariable>) => value?.value === '$__all'
+              );
+              if (isAll) {
+                return onUpdate(selectableOptions);
+              }
 
-            onUpdate(selected);
-          }}
-          value={options?.filter(
-            (option: SelectableValue<SelectableVariable>) =>
-              !!selectedOptions?.find((selected: string) => selected === option?.value?.value)
-          )}
-          filterOption={(option: SelectableValue<SelectableVariable>, searchQuery: string) =>
-            !!option?.label?.toLowerCase().includes(searchQuery.toLowerCase())
-          }
-          closeMenuOnSelect={false}
-          options={options}
-        />
-      )}
+              onUpdate(selected);
+            }}
+            value={options?.filter(
+              (option: SelectableValue<SelectableVariable>) =>
+                !!selectedOptions?.find((selected: string) => selected === option?.value?.value)
+            )}
+            filterOption={(option: SelectableValue<SelectableVariable>, searchQuery: string) =>
+              !!option?.label?.toLowerCase().includes(searchQuery.toLowerCase())
+            }
+            closeMenuOnSelect={false}
+            options={options}
+          />
+        )}
+      </div>
```

Comment:
Same as above, bit hard to parse the details of these transformation functions (and again some possible missed opportunities for memoisation)?